models/ACPO-dummy-inliner-model-gen.py

```diff
@@ -1,4 +1,4 @@
-#===- ACPO-dummy-inliner-model-gen.py - Generate Loop Unroll Model ------------===//
+#===- ACPO-dummy-inliner-model-gen.py - Generate Inliner Dummy Model -----===//
```

Comment:
I suggest adopting the style used in other Python scripts in the LLVM source tree (e.g. `compiler-rt/lib/asan/scripts/asan_symbolize.py`, `.github/workflows/commit-access-review.py`):
```python
#!/usr/bin/env python3
# ===- ACPO-dummy-inliner-model-gen.py - Generate Dummy Inliner Model ------===#
#
# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
# See https://llvm.org/LICENSE.txt for license information.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
#
# ===-----------------------------------------------------------------------===#
```

Reply:
Thanks for the suggestion. Repushed!